api/config/config_stable.json

```diff
@@ -94,6 +94,9 @@
     "host": "aou.nnlm.gov",
     "credentialsKeyV2": "moodle-prod-key-v2.txt"
   },
+  "absorb": {
+    "externalDepartmentId": "Development"
```

Comment:
In the stable env we use prod Moodle. However, the stable env users are in the development GSuite, so I think it's easier to use Absorb's development department rather than add special logic here. Let me know if there's a reason we should be using the Absorb prod department instead.